src/test/daml/Daml/Finance/Util/Test/Date/Calendar.daml

```diff
@@ -50,8 +50,8 @@ test_addBusinessDays = script do
 
 test_adjustDate : Script ()
 test_adjustDate = script do
-  adjustDate cal NONE (date 2018 Mar 31) === (date 2018 Mar 31)
-  adjustDate cal FOLLOWING (date 2018 Mar 31) === (date 2018 Apr 02)
-  adjustDate cal MODFOLLOWING (date 2018 Mar 31) === (date 2018 Mar 30)
-  adjustDate cal PRECEDING (date 2018 Apr 01) === (date 2018 Mar 30)
-  adjustDate cal MODPRECEDING (date 2018 Apr 01) === (date 2018 Apr 02)
+  adjustDate cal Unadjusted (date 2018 Mar 31) === (date 2018 Mar 31)
```

Comment:
I think I prefer None over Unadjusted, since it specifies which adjustment rule should be applied to the input dates (which are unadjusted).

This also matches the ISDA CDM, see below for reference:

-- | The enumerated values to specify the convention for
--   adjusting any relevant date if it would otherwise
--   fall on a day that is not a valid business day.
data BusinessDayConventionEnum 
  = BusinessDayConventionEnum_FOLLOWING
  -- ^ The non-business date will be adjusted to the first
  --   following day that is a business day
  | BusinessDayConventionEnum_FRN
  -- ^ Per 2000 ISDA Definitions, Section 4.11. FRN
  --   Convention; Eurodollar Convention. FRN is included
  --   here as a type of business day convention although it
  --   does not strictly fall within ISDA&#39;s definition
  --   of a Business Day Convention and does not conform to
  --   the simple definition given above.
  | BusinessDayConventionEnum_MODFOLLOWING
  -- ^ The non-business date will be adjusted to the first
  --   following day that is a business day unless that day
  --   falls in the next calendar month, in which case that
  --   date will be the first preceding day that is a
  --   business day.
  | BusinessDayConventionEnum_MODPRECEDING
  -- ^ The non-business date will be adjusted to the first
  --   preceding day that is a business day unless that day
  --   falls in the previous calendar month, in which case
  --   that date will be the first following day that us a
  --   business day.
  | BusinessDayConventionEnum_NEAREST
  -- ^ The non-business date will be adjusted to the nearest
  --   day that is a business day - i.e. if the non-business
  --   day falls on any day other than a Sunday or a Monday,
  --   it will be the first preceding day that is a business
  --   day, and will be the first following business day if
  --   it falls on a Sunday or a Monday.
  | BusinessDayConventionEnum_NONE
  -- ^ The date will not be adjusted if it falls on a day
  --   that is not a business day.
  | BusinessDayConventionEnum_NotApplicable
  -- ^ The date adjustments conventions are defined
  --   elsewhere, so it is not required to specify them
  --   here.
  | BusinessDayConventionEnum_PRECEDING
  -- ^ The non-business day will be adjusted to the first
  --   preceding day that is a business day.
    deriving (Eq, Ord, Show)

Reply:
Changed to `NoAdjustment` as discussed (`None` is not ideal as it clases with Optional's 'None')